ixmp4/conf/settings.py

```diff
@@ -144,8 +145,10 @@ def configure_logging(self, config: str):
         os.environ.setdefault("IXMP4_DEBUG_LOG", str(debug_file.absolute()))
         os.environ.setdefault("IXMP4_ERROR_LOG", str(error_file.absolute()))
 
-        logging_config = here / f"logging/{config}.conf"
-        logging.config.fileConfig(logging_config, disable_existing_loggers=False)
+        logging_config = here / f"logging/{config}.json"
+        with open(logging_config, "r") as file:
```

Comment:
Suggest shorten for simplicity
```suggestion
        with open(logging_config) as file:
```